go.mod

```diff
@@ -118,7 +121,8 @@ replace (
 
 	kubevirt.io/client-go => ./staging/src/kubevirt.io/client-go
 
-	kubevirt.io/containerized-data-importer => kubevirt.io/containerized-data-importer v1.36.0
+	kubevirt.io/containerized-data-importer => kubevirt.io/containerized-data-importer v1.41.0
```

Comment:
Do we also need real code from CDI? Or is this just an artifact from our go.mod file in staging/client-go/go.mod?

Reply:
Yeah, unfortunately there is other code imported from CDI.  A function for authorizing clone DataVolumes.

https://github.com/kubevirt/containerized-data-importer/tree/main/pkg/clone

Reply:
Ok, thanks!.